zerver/lib/email_notifications.py

```diff
@@ -700,6 +702,50 @@ def followup_email_delay(user: UserProfile, *, days: int) -> timedelta:
     return timedelta(days=days_to_delay, hours=-1)
 
 
+FOLLOWUP_ZULIP_GUIDE_CONTEXT: Dict[int, Dict[str, Any]] = {
+    Realm.ORG_TYPES["business"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for businesses"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for businesses"),
+        "for_zulip_link": "https://zulip.com/for/business",
+    },
+    Realm.ORG_TYPES["opensource"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for open-source projects"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for open-source projects"),
+        "for_zulip_link": "https://zulip.com/for/open-source",
+    },
+    Realm.ORG_TYPES["education_nonprofit"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for education"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for education"),
+        "for_zulip_link": "https://zulip.com/for/education",
+    },
+    Realm.ORG_TYPES["education"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for education"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for education"),
+        "for_zulip_link": "https://zulip.com/for/education",
+    },
+    Realm.ORG_TYPES["research"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for research"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for research"),
+        "for_zulip_link": "https://zulip.com/for/research",
+    },
+    Realm.ORG_TYPES["event"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for events and conferences"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for events and conferences"),
+        "for_zulip_link": "https://zulip.com/for/events",
+    },
+    Realm.ORG_TYPES["nonprofit"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for non-profits"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for non-profits"),
+        "for_zulip_link": "https://zulip.com/for/communities",
+    },
+    Realm.ORG_TYPES["community"]["id"]: {
+        "zulip_guide_subject": gettext_lazy("Zulip guide for communities"),
+        "zulip_guide_button_text": gettext_lazy("View Zulip guide for communities"),
+        "for_zulip_link": "https://zulip.com/for/communities",
+    },
+}
```

Comment:
Maybe add a comment here just documenting the expected behavior if something doesn't appear here, and/or how to decide whether to add something to this list?

Reply:
Updated version does not have this dict structure any more. Instead, the `/for/.../` page url is stored in the `Realm.ORG_TYPES` dict in `models.py`.